diff -Naur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/acinclude.m4 neutrino-hd2-exp/acinclude.m4
--- neutrino-hd2-exp.org/acinclude.m4	2012-12-17 13:14:29.637472913 +0100
+++ neutrino-hd2-exp/acinclude.m4	2012-12-17 13:14:29.769472918 +0100
@@ -225,10 +225,10 @@
 AC_DEFUN([_TUXBOX_APPS_LIB_PKGCONFIG],[
 AC_REQUIRE([TUXBOX_APPS_PKGCONFIG])
 AC_MSG_CHECKING(for package $2)
-if PKG_CONFIG_PATH="${prefix}/lib/pkgconfig" $PKG_CONFIG --exists "$2" ; then
+if $PKG_CONFIG --exists "$2" ; then
 	AC_MSG_RESULT(yes)
-	$1_CFLAGS=$(PKG_CONFIG_PATH="${prefix}/lib/pkgconfig" $PKG_CONFIG --cflags "$2")
-	$1_LIBS=$(PKG_CONFIG_PATH="${prefix}/lib/pkgconfig" $PKG_CONFIG --libs "$2")
+	$1_CFLAGS=$($PKG_CONFIG --cflags "$2")
+	$1_LIBS=$($PKG_CONFIG --libs "$2")
 else
 	AC_MSG_RESULT(no)
 fi
diff -Naur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/lib/libnet/network_interfaces.cpp neutrino-hd2-exp/lib/libnet/network_interfaces.cpp
--- neutrino-hd2-exp.org/lib/libnet/network_interfaces.cpp	2012-12-17 13:14:11.329472154 +0100
+++ neutrino-hd2-exp/lib/libnet/network_interfaces.cpp	2012-12-17 13:14:29.769472918 +0100
@@ -24,6 +24,7 @@
 #include <map>
 #include <string>
 #include <sstream>
+#include <unistd.h>
 
 #include <unistd.h>
 
diff -Naur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/driver/framebuffer.h neutrino-hd2-exp/src/driver/framebuffer.h
--- neutrino-hd2-exp.org/src/driver/framebuffer.h	2012-12-17 13:14:22.985472637 +0100
+++ neutrino-hd2-exp/src/driver/framebuffer.h	2012-12-17 13:14:29.769472918 +0100
@@ -55,11 +55,6 @@
 #define CORNER_BOTTOM		0x2
 #define CORNER_BOTH		0x3
 
-// FB HW BLIT
-#if !defined (PLATFORM_GENERIC)
-#define FB_BLIT
-#endif
-
 // resolution
 #define DEFAULT_XRES		960
 #define DEFAULT_YRES		720
diff -Naur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/driver/gif.cpp neutrino-hd2-exp/src/driver/gif.cpp
--- neutrino-hd2-exp.org/src/driver/gif.cpp	2012-12-17 13:14:22.985472637 +0100
+++ neutrino-hd2-exp/src/driver/gif.cpp	2012-12-17 13:14:29.769472918 +0100
@@ -4,7 +4,7 @@
 
 #include <cstring>
 #include <cstdlib>
-
+#include <unistd.h>
 #include "framebuffer.h"
 #include <stdio.h>
 #include <sys/types.h>
diff -Naur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/driver/nglcd.cpp neutrino-hd2-exp/src/driver/nglcd.cpp
--- neutrino-hd2-exp.org/src/driver/nglcd.cpp	2012-12-17 13:14:22.993472637 +0100
+++ neutrino-hd2-exp/src/driver/nglcd.cpp	2012-12-17 13:14:29.769472918 +0100
@@ -32,7 +32,7 @@
 #include <algorithm>
 #include <system/debug.h>
 #include <system/set_threadname.h>
-
+#include <unistd.h>
 #include "nglcd.h"
 
 #ifndef max
diff -Naur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/gui/audioplayer.cpp neutrino-hd2-exp/src/gui/audioplayer.cpp
--- neutrino-hd2-exp.org/src/gui/audioplayer.cpp	2012-12-17 13:14:19.081472475 +0100
+++ neutrino-hd2-exp/src/gui/audioplayer.cpp	2012-12-17 13:14:29.769472918 +0100
@@ -74,6 +74,7 @@
 #include <fstream>
 #include <iostream>
 #include <sstream>
+#include <unistd.h>
 
 #include <gui/pictureviewer.h>
 #include <audio_cs.h>
diff -Naur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/gui/filebrowser.cpp neutrino-hd2-exp/src/gui/filebrowser.cpp
--- neutrino-hd2-exp.org/src/gui/filebrowser.cpp	2012-12-17 13:14:19.085472475 +0100
+++ neutrino-hd2-exp/src/gui/filebrowser.cpp	2012-12-17 13:14:29.769472918 +0100
@@ -57,6 +57,7 @@
 #include <stdlib.h>
 #include <dirent.h>
 #include <sstream>
+#include <unistd.h>
 
 #include <sys/stat.h>
 
diff -Naur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/gui/moviebrowser.cpp neutrino-hd2-exp/src/gui/moviebrowser.cpp
--- neutrino-hd2-exp.org/src/gui/moviebrowser.cpp	2012-12-17 13:14:19.077472475 +0100
+++ neutrino-hd2-exp/src/gui/moviebrowser.cpp	2012-12-17 13:14:29.769472918 +0100
@@ -69,7 +69,7 @@
 #include <utime.h>
 #include <gui/widget/progressbar.h>
 #include <gui/pictureviewer.h>
-
+#include <unistd.h>
 #include <system/debug.h>
 
 #include <driver/vcrcontrol.h>
diff -Naur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/gui/movieinfo.cpp neutrino-hd2-exp/src/gui/movieinfo.cpp
--- neutrino-hd2-exp.org/src/gui/movieinfo.cpp	2012-12-17 13:14:19.077472475 +0100
+++ neutrino-hd2-exp/src/gui/movieinfo.cpp	2012-12-17 13:14:29.769472918 +0100
@@ -54,6 +54,7 @@
 
 #include <gui/widget/msgbox.h>
 #include <gui/movieinfo.h>
+#include <unistd.h>
 
 //#define XMLTREE_LIB
 #ifdef XMLTREE_LIB
diff -Naur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/gui/opkg_manager.cpp neutrino-hd2-exp/src/gui/opkg_manager.cpp
--- neutrino-hd2-exp.org/src/gui/opkg_manager.cpp	2012-12-17 13:14:19.077472475 +0100
+++ neutrino-hd2-exp/src/gui/opkg_manager.cpp	2012-12-17 13:14:29.773472918 +0100
@@ -55,7 +55,7 @@
 #include <system/debug.h>
 
 #include <stdio.h>
-
+#include <unistd.h>
 
 COPKGManager::COPKGManager()
 {
diff -Naur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/nhttpd/tuxboxapi/neutrinoapi.cpp neutrino-hd2-exp/src/nhttpd/tuxboxapi/neutrinoapi.cpp
--- neutrino-hd2-exp.org/src/nhttpd/tuxboxapi/neutrinoapi.cpp	2012-12-17 13:14:11.581472164 +0100
+++ neutrino-hd2-exp/src/nhttpd/tuxboxapi/neutrinoapi.cpp	2012-12-17 13:14:29.773472918 +0100
@@ -17,6 +17,7 @@
 #include <fstream>
 #include <map>
 #include <sstream>
+#include <unistd.h>
 
 #include <unistd.h>
 
diff -Naur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/nhttpd/web/scripts/Makefile.am neutrino-hd2-exp/src/nhttpd/web/scripts/Makefile.am
--- neutrino-hd2-exp.org/src/nhttpd/web/scripts/Makefile.am	2012-12-17 13:14:12.177472189 +0100
+++ neutrino-hd2-exp/src/nhttpd/web/scripts/Makefile.am	2012-12-17 13:14:29.773472918 +0100
@@ -3,5 +3,5 @@
 install_DATA= api.sh _Y_Globals.sh _Y_Library.sh Y_Live.sh Y_Tools.sh
 
 install-data-hook:
-	chmod 0755 $(DATADIR)/neutrino/httpd/scripts/api.sh
-	chmod 0755 $(DATADIR)/neutrino/httpd/scripts/Y_*.sh
+	chmod 0755 $(DESTDIR)$(DATADIR)/neutrino/httpd/scripts/api.sh
+	chmod 0755 $(DESTDIR)$(DATADIR)/neutrino/httpd/scripts/Y_*.sh
diff -Naur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/nhttpd/yhttpd_core/yrequest.cpp neutrino-hd2-exp/src/nhttpd/yhttpd_core/yrequest.cpp
--- neutrino-hd2-exp.org/src/nhttpd/yhttpd_core/yrequest.cpp	2012-12-17 13:14:11.873472176 +0100
+++ neutrino-hd2-exp/src/nhttpd/yhttpd_core/yrequest.cpp	2012-12-17 13:14:29.773472918 +0100
@@ -21,6 +21,8 @@
 #include <cstring>
 #include <cstdlib>
 #include <errno.h>
+#include <unistd.h>
+
 // system
 #include <fcntl.h>
 #include <sys/socket.h>
diff -Naur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/nhttpd/yhttpd_core/yresponse.cpp neutrino-hd2-exp/src/nhttpd/yhttpd_core/yresponse.cpp
--- neutrino-hd2-exp.org/src/nhttpd/yhttpd_core/yresponse.cpp	2012-12-17 13:14:11.873472176 +0100
+++ neutrino-hd2-exp/src/nhttpd/yhttpd_core/yresponse.cpp	2012-12-17 13:14:29.773472918 +0100
@@ -9,6 +9,7 @@
 #include <errno.h>
 // c++
 #include <string.h>
+#include <unistd.h>
 // system
 #include <fcntl.h>
 #include <sys/socket.h>
diff -Naur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/nhttpd/yhttpd_core/ysocket.cpp neutrino-hd2-exp/src/nhttpd/yhttpd_core/ysocket.cpp
--- neutrino-hd2-exp.org/src/nhttpd/yhttpd_core/ysocket.cpp	2012-12-17 13:14:11.873472176 +0100
+++ neutrino-hd2-exp/src/nhttpd/yhttpd_core/ysocket.cpp	2012-12-17 13:14:29.773472918 +0100
@@ -5,7 +5,7 @@
 
 #include <cstring>
 #include <cstdio>
-
+#include <unistd.h>
 // system
 #include <arpa/inet.h>
 #include <netinet/tcp.h>
diff -Naur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/nhttpd/yhttpd_mods/mod_weblog.cpp neutrino-hd2-exp/src/nhttpd/yhttpd_mods/mod_weblog.cpp
--- neutrino-hd2-exp.org/src/nhttpd/yhttpd_mods/mod_weblog.cpp	2012-12-17 13:14:15.513472328 +0100
+++ neutrino-hd2-exp/src/nhttpd/yhttpd_mods/mod_weblog.cpp	2012-12-17 13:14:29.773472918 +0100
@@ -5,7 +5,7 @@
 // Normally
 //=============================================================================
 #include <stdio.h>
-
+#include <unistd.h>
 #include <cstdarg>
 #include <cstring>
 #include <cstdlib>
diff -Naur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/system/configure_network.cpp neutrino-hd2-exp/src/system/configure_network.cpp
--- neutrino-hd2-exp.org/src/system/configure_network.cpp	2012-12-17 13:14:19.797472505 +0100
+++ neutrino-hd2-exp/src/system/configure_network.cpp	2012-12-17 13:14:29.773472918 +0100
@@ -36,7 +36,7 @@
 #include <unistd.h>
 
 #include <system/safe_system.h>
-
+#include <unistd.h>
 
 CNetworkConfig::CNetworkConfig(void)
 {
diff -Naur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/zapit/src/getservices.cpp neutrino-hd2-exp/src/zapit/src/getservices.cpp
--- neutrino-hd2-exp.org/src/zapit/src/getservices.cpp	2012-12-17 13:14:21.017472555 +0100
+++ neutrino-hd2-exp/src/zapit/src/getservices.cpp	2012-12-17 13:14:29.773472918 +0100
@@ -32,7 +32,7 @@
 #include <set>
 #include <sys/stat.h>
 #include <sys/vfs.h>
-
+#include <unistd.h>
 #include <errno.h>
 
 #include <unistd.h>
diff -Naur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/zapit/src/nit.cpp neutrino-hd2-exp/src/zapit/src/nit.cpp
--- neutrino-hd2-exp.org/src/zapit/src/nit.cpp	2012-12-17 13:14:21.017472555 +0100
+++ neutrino-hd2-exp/src/zapit/src/nit.cpp	2012-12-17 13:14:29.773472918 +0100
@@ -27,6 +27,7 @@
 #include <cerrno>
 #include <cstdio>
 #include <cstring>
+#include <unistd.h>
 
 #include <unistd.h>
 
