diff -Naur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/acinclude.m4 neutrino-hd2-exp/acinclude.m4
--- neutrino-hd2-exp.org/acinclude.m4	2012-12-17 13:14:29.637472913 +0100
+++ neutrino-hd2-exp/acinclude.m4	2012-12-17 13:14:29.769472918 +0100
@@ -225,10 +225,10 @@
 AC_DEFUN([_TUXBOX_APPS_LIB_PKGCONFIG],[
 AC_REQUIRE([TUXBOX_APPS_PKGCONFIG])
 AC_MSG_CHECKING(for package $2)
-if PKG_CONFIG_PATH="${prefix}/lib/pkgconfig" $PKG_CONFIG --exists "$2" ; then
+if $PKG_CONFIG --exists "$2" ; then
 	AC_MSG_RESULT(yes)
-	$1_CFLAGS=$(PKG_CONFIG_PATH="${prefix}/lib/pkgconfig" $PKG_CONFIG --cflags "$2")
-	$1_LIBS=$(PKG_CONFIG_PATH="${prefix}/lib/pkgconfig" $PKG_CONFIG --libs "$2")
+	$1_CFLAGS=$($PKG_CONFIG --cflags "$2")
+	$1_LIBS=$($PKG_CONFIG --libs "$2")
 else
 	AC_MSG_RESULT(no)
 fi
diff -Naur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/driver/framebuffer.h neutrino-hd2-exp/src/driver/framebuffer.h
--- neutrino-hd2-exp.org/src/driver/framebuffer.h	2012-12-17 13:14:22.985472637 +0100
+++ neutrino-hd2-exp/src/driver/framebuffer.h	2012-12-17 13:14:29.769472918 +0100
@@ -55,11 +55,6 @@
 #define CORNER_BOTTOM		0x2
 #define CORNER_BOTH		0x3
 
-// FB HW BLIT
-#if !defined (PLATFORM_GENERIC)
-#define FB_BLIT
-#endif
-
 // resolution
 #define DEFAULT_XRES		960
 #define DEFAULT_YRES		720
diff -Naur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/nhttpd/web/scripts/Makefile.am neutrino-hd2-exp/src/nhttpd/web/scripts/Makefile.am
--- neutrino-hd2-exp.org/src/nhttpd/web/scripts/Makefile.am	2012-12-17 13:14:12.177472189 +0100
+++ neutrino-hd2-exp/src/nhttpd/web/scripts/Makefile.am	2012-12-17 13:14:29.773472918 +0100
@@ -3,5 +3,5 @@
 install_DATA= api.sh _Y_Globals.sh _Y_Library.sh Y_Live.sh Y_Tools.sh
 
 install-data-hook:
-	chmod 0755 $(DATADIR)/neutrino/httpd/scripts/api.sh
-	chmod 0755 $(DATADIR)/neutrino/httpd/scripts/Y_*.sh
+	chmod 0755 $(DESTDIR)$(DATADIR)/neutrino/httpd/scripts/api.sh
+	chmod 0755 $(DESTDIR)$(DATADIR)/neutrino/httpd/scripts/Y_*.sh

