diff -Nur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/acinclude.m4 neutrino-hd2-exp/acinclude.m4
--- neutrino-hd2-exp.org/acinclude.m4	2012-12-02 23:13:34.052785833 +0100
+++ neutrino-hd2-exp/acinclude.m4	2012-12-02 23:13:34.196785839 +0100
@@ -225,10 +225,10 @@
 AC_DEFUN([_TUXBOX_APPS_LIB_PKGCONFIG],[
 AC_REQUIRE([TUXBOX_APPS_PKGCONFIG])
 AC_MSG_CHECKING(for package $2)
-if PKG_CONFIG_PATH="${prefix}/lib/pkgconfig" $PKG_CONFIG --exists "$2" ; then
+if $PKG_CONFIG --exists "$2" ; then
 	AC_MSG_RESULT(yes)
-	$1_CFLAGS=$(PKG_CONFIG_PATH="${prefix}/lib/pkgconfig" $PKG_CONFIG --cflags "$2")
-	$1_LIBS=$(PKG_CONFIG_PATH="${prefix}/lib/pkgconfig" $PKG_CONFIG --libs "$2")
+	$1_CFLAGS=$($PKG_CONFIG --cflags "$2")
+	$1_LIBS=$($PKG_CONFIG --libs "$2")
 else
 	AC_MSG_RESULT(no)
 fi
diff -Nur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/lib/libcoolstream/dvb-ci.cpp neutrino-hd2-exp/lib/libcoolstream/dvb-ci.cpp
--- neutrino-hd2-exp.org/lib/libcoolstream/dvb-ci.cpp	2012-11-13 22:59:52.084427239 +0100
+++ neutrino-hd2-exp/lib/libcoolstream/dvb-ci.cpp	2012-11-13 23:06:58.096444897 +0100
@@ -6,7 +6,7 @@
 #include <fcntl.h>
 #include <sys/ioctl.h>
 #include <asm/types.h>
-
+#include <unistd.h>
 #include <linux/dvb/ca.h>
 #include <poll.h>
  
diff -Nur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/lib/libcoolstream/record_cs.cpp neutrino-hd2-exp/lib/libcoolstream/record_cs.cpp
--- neutrino-hd2-exp.org/lib/libcoolstream/record_cs.cpp	2012-11-13 22:59:52.084427239 +0100
+++ neutrino-hd2-exp/lib/libcoolstream/record_cs.cpp	2012-11-13 23:05:13.516440561 +0100
@@ -21,7 +21,7 @@
 #include <stdio.h>
 #include <stdint.h>
 #include <string.h> 
-
+#include <unistd.h>
 #include <sys/types.h>
 
 #include "record_cs.h"
diff -Nur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/lib/libnet/network_interfaces.cpp neutrino-hd2-exp/lib/libnet/network_interfaces.cpp
--- neutrino-hd2-exp.org/lib/libnet/network_interfaces.cpp	2012-12-02 23:13:20.656785278 +0100
+++ neutrino-hd2-exp/lib/libnet/network_interfaces.cpp	2012-12-02 23:13:34.196785839 +0100
@@ -24,6 +24,7 @@
 #include <map>
 #include <string>
 #include <sstream>
+#include <unistd.h>
 
 /*
  * Known bugs:
diff -Nur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/driver/framebuffer.h neutrino-hd2-exp/src/driver/framebuffer.h
--- neutrino-hd2-exp.org/src/driver/framebuffer.h	2012-12-02 23:13:27.412785559 +0100
+++ neutrino-hd2-exp/src/driver/framebuffer.h	2012-12-02 23:13:34.196785839 +0100
@@ -55,11 +55,6 @@
 #define CORNER_BOTTOM		0x2
 #define CORNER_BOTH		0x3
 
-// FB HW BLIT
-#if !defined (PLATFORM_GENERIC)
-#define FB_BLIT
-#endif
-
 // resolution
 #define DEFAULT_XRES		960
 #define DEFAULT_YRES		720
diff -Nur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/driver/gif.cpp neutrino-hd2-exp/src/driver/gif.cpp
--- neutrino-hd2-exp.org/src/driver/gif.cpp	2012-12-02 23:13:27.408785559 +0100
+++ neutrino-hd2-exp/src/driver/gif.cpp	2012-12-02 23:13:34.196785839 +0100
@@ -4,7 +4,7 @@
 
 #include <cstring>
 #include <cstdlib>
-
+#include <unistd.h>
 #include "framebuffer.h"
 #include <stdio.h>
 #include <sys/types.h>
diff -Nur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/driver/nglcd.cpp neutrino-hd2-exp/src/driver/nglcd.cpp
--- neutrino-hd2-exp.org/src/driver/nglcd.cpp	2012-12-02 23:13:27.416785559 +0100
+++ neutrino-hd2-exp/src/driver/nglcd.cpp	2012-12-02 23:13:34.196785839 +0100
@@ -29,7 +29,7 @@
 #include <algorithm>
 #include <system/debug.h>
 #include <system/set_threadname.h>
-
+#include <unistd.h>
 #include "nglcd.h"
 
 #ifndef max
diff -Nur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/driver/pictureviewer/pictureviewer.cpp neutrino-hd2-exp/src/driver/pictureviewer/pictureviewer.cpp
--- neutrino-hd2-exp.org/src/driver/pictureviewer/pictureviewer.cpp	2012-12-02 23:13:27.068785544 +0100
+++ neutrino-hd2-exp/src/driver/pictureviewer/pictureviewer.cpp	2012-12-02 23:13:34.196785839 +0100
@@ -8,7 +8,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <sys/types.h>
-
+#include <unistd.h>
 #include <system/debug.h>
 
 
diff -Nur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/driver/shutdown_count.cpp neutrino-hd2-exp/src/driver/shutdown_count.cpp
--- neutrino-hd2-exp.org/src/driver/shutdown_count.cpp	2012-12-02 23:13:27.412785559 +0100
+++ neutrino-hd2-exp/src/driver/shutdown_count.cpp	2012-12-02 23:13:34.196785839 +0100
@@ -34,7 +34,7 @@
 
 #include <sys/timeb.h>
 #include <time.h>
-
+#include <unistd.h>
 
 SHTDCNT::SHTDCNT()
 	: configfile('\t')
diff -Nur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/gui/audioplayer.cpp neutrino-hd2-exp/src/gui/audioplayer.cpp
--- neutrino-hd2-exp.org/src/gui/audioplayer.cpp	2012-12-02 23:13:24.744785447 +0100
+++ neutrino-hd2-exp/src/gui/audioplayer.cpp	2012-12-02 23:13:34.196785839 +0100
@@ -72,6 +72,7 @@
 #include <fstream>
 #include <iostream>
 #include <sstream>
+#include <unistd.h>
 
 #include <gui/pictureviewer.h>
 #include <audio_cs.h>
diff -Nur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/gui/filebrowser.cpp neutrino-hd2-exp/src/gui/filebrowser.cpp
--- neutrino-hd2-exp.org/src/gui/filebrowser.cpp	2012-12-02 23:13:24.744785447 +0100
+++ neutrino-hd2-exp/src/gui/filebrowser.cpp	2012-12-02 23:13:34.196785839 +0100
@@ -55,6 +55,7 @@
 #include <stdlib.h>
 #include <dirent.h>
 #include <sstream>
+#include <unistd.h>
 
 #include <sys/stat.h>
 
diff -Nur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/gui/moviebrowser.cpp neutrino-hd2-exp/src/gui/moviebrowser.cpp
--- neutrino-hd2-exp.org/src/gui/moviebrowser.cpp	2012-12-02 23:13:24.736785447 +0100
+++ neutrino-hd2-exp/src/gui/moviebrowser.cpp	2012-12-02 23:13:34.196785839 +0100
@@ -66,7 +66,7 @@
 #include <utime.h>
 #include <gui/widget/progressbar.h>
 #include <gui/pictureviewer.h>
-
+#include <unistd.h>
 #include <system/debug.h>
 
 #include <driver/vcrcontrol.h>
diff -Nur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/gui/movieinfo.cpp neutrino-hd2-exp/src/gui/movieinfo.cpp
--- neutrino-hd2-exp.org/src/gui/movieinfo.cpp	2012-12-02 23:13:24.736785447 +0100
+++ neutrino-hd2-exp/src/gui/movieinfo.cpp	2012-12-02 23:13:34.196785839 +0100
@@ -51,6 +51,7 @@
 #include <sys/types.h>
 #include <gui/widget/msgbox.h>
 #include <gui/movieinfo.h>
+#include <unistd.h>
 
 //#define XMLTREE_LIB
 #ifdef XMLTREE_LIB
diff -Nur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/gui/opkg_manager.cpp neutrino-hd2-exp/src/gui/opkg_manager.cpp
--- neutrino-hd2-exp.org/src/gui/opkg_manager.cpp	2012-12-02 23:13:24.736785447 +0100
+++ neutrino-hd2-exp/src/gui/opkg_manager.cpp	2012-12-02 23:13:34.196785839 +0100
@@ -54,7 +54,7 @@
 #include <system/debug.h>
 
 #include <stdio.h>
-
+#include <unistd.h>
 
 COPKGManager::COPKGManager()
 {
diff -Nur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/nhttpd/tuxboxapi/neutrinoapi.cpp neutrino-hd2-exp/src/nhttpd/tuxboxapi/neutrinoapi.cpp
--- neutrino-hd2-exp.org/src/nhttpd/tuxboxapi/neutrinoapi.cpp	2012-12-02 23:13:20.812785284 +0100
+++ neutrino-hd2-exp/src/nhttpd/tuxboxapi/neutrinoapi.cpp	2012-12-02 23:13:34.200785839 +0100
@@ -17,6 +17,7 @@
 #include <fstream>
 #include <map>
 #include <sstream>
+#include <unistd.h>
 
 // tuxbox
 #include <neutrinoMessages.h>
diff -Nur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/nhttpd/web/scripts/Makefile.am neutrino-hd2-exp/src/nhttpd/web/scripts/Makefile.am
--- neutrino-hd2-exp.org/src/nhttpd/web/scripts/Makefile.am	2012-12-02 23:13:21.212785301 +0100
+++ neutrino-hd2-exp/src/nhttpd/web/scripts/Makefile.am	2012-12-02 23:13:34.200785839 +0100
@@ -3,5 +3,5 @@
 install_DATA= api.sh _Y_Globals.sh _Y_Library.sh Y_Live.sh Y_Tools.sh
 
 install-data-hook:
-	chmod 0755 $(DATADIR)/neutrino/httpd/scripts/api.sh
-	chmod 0755 $(DATADIR)/neutrino/httpd/scripts/Y_*.sh
+	chmod 0755 $(DESTDIR)$(DATADIR)/neutrino/httpd/scripts/api.sh
+	chmod 0755 $(DESTDIR)$(DATADIR)/neutrino/httpd/scripts/Y_*.sh
diff -Nur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/nhttpd/yhttpd_core/ylanguage.cpp neutrino-hd2-exp/src/nhttpd/yhttpd_core/ylanguage.cpp
--- neutrino-hd2-exp.org/src/nhttpd/yhttpd_core/ylanguage.cpp	2012-12-02 23:13:20.956785291 +0100
+++ neutrino-hd2-exp/src/nhttpd/yhttpd_core/ylanguage.cpp	2012-12-02 23:13:34.200785839 +0100
@@ -7,7 +7,7 @@
 #include <cstdarg>
 #include <cstdio>
 #include <cstdlib>
-
+#include <unistd.h>
 // yhttpd
 #include "yconfig.h"
 #include "yhttpd.h"
diff -Nur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/nhttpd/yhttpd_core/yrequest.cpp neutrino-hd2-exp/src/nhttpd/yhttpd_core/yrequest.cpp
--- neutrino-hd2-exp.org/src/nhttpd/yhttpd_core/yrequest.cpp	2012-12-02 23:13:20.956785291 +0100
+++ neutrino-hd2-exp/src/nhttpd/yhttpd_core/yrequest.cpp	2012-12-02 23:13:34.200785839 +0100
@@ -21,6 +21,8 @@
 #include <cstring>
 #include <cstdlib>
 #include <errno.h>
+#include <unistd.h>
+
 // system
 #include <fcntl.h>
 #include <sys/socket.h>
diff -Nur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/nhttpd/yhttpd_core/yresponse.cpp neutrino-hd2-exp/src/nhttpd/yhttpd_core/yresponse.cpp
--- neutrino-hd2-exp.org/src/nhttpd/yhttpd_core/yresponse.cpp	2012-12-02 23:13:20.956785291 +0100
+++ neutrino-hd2-exp/src/nhttpd/yhttpd_core/yresponse.cpp	2012-12-02 23:13:34.200785839 +0100
@@ -9,6 +9,7 @@
 #include <errno.h>
 // c++
 #include <string.h>
+#include <unistd.h>
 // system
 #include <fcntl.h>
 #include <sys/socket.h>
diff -Nur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/nhttpd/yhttpd_core/ysocket.cpp neutrino-hd2-exp/src/nhttpd/yhttpd_core/ysocket.cpp
--- neutrino-hd2-exp.org/src/nhttpd/yhttpd_core/ysocket.cpp	2012-12-02 23:13:20.956785291 +0100
+++ neutrino-hd2-exp/src/nhttpd/yhttpd_core/ysocket.cpp	2012-12-02 23:13:34.200785839 +0100
@@ -5,7 +5,7 @@
 
 #include <cstring>
 #include <cstdio>
-
+#include <unistd.h>
 // system
 #include <arpa/inet.h>
 #include <netinet/tcp.h>
diff -Nur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/nhttpd/yhttpd_mods/mod_sendfile.cpp neutrino-hd2-exp/src/nhttpd/yhttpd_mods/mod_sendfile.cpp
--- neutrino-hd2-exp.org/src/nhttpd/yhttpd_mods/mod_sendfile.cpp	2012-12-02 23:13:22.516785355 +0100
+++ neutrino-hd2-exp/src/nhttpd/yhttpd_mods/mod_sendfile.cpp	2012-12-02 23:13:34.200785839 +0100
@@ -45,6 +45,7 @@
 #include <fcntl.h>
 #include <pthread.h>
 #include <sys/stat.h>
+#include <unistd.h>
 // yhttpd
 #include "yconfig.h"
 #include "ytypes_globals.h"
diff -Nur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/nhttpd/yhttpd_mods/mod_weblog.cpp neutrino-hd2-exp/src/nhttpd/yhttpd_mods/mod_weblog.cpp
--- neutrino-hd2-exp.org/src/nhttpd/yhttpd_mods/mod_weblog.cpp	2012-12-02 23:13:22.516785355 +0100
+++ neutrino-hd2-exp/src/nhttpd/yhttpd_mods/mod_weblog.cpp	2012-12-02 23:13:34.200785839 +0100
@@ -5,7 +5,7 @@
 // Normally
 //=============================================================================
 #include <stdio.h>
-
+#include <unistd.h>
 #include <cstdarg>
 #include <cstring>
 #include <cstdlib>
diff -Nur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/system/configure_network.cpp neutrino-hd2-exp/src/system/configure_network.cpp
--- neutrino-hd2-exp.org/src/system/configure_network.cpp	2012-12-02 23:13:25.132785462 +0100
+++ neutrino-hd2-exp/src/system/configure_network.cpp	2012-12-02 23:13:34.200785839 +0100
@@ -33,7 +33,7 @@
 #include <sstream>
 #include <fstream>
 #include <system/safe_system.h>
-
+#include <unistd.h>
 
 CNetworkConfig::CNetworkConfig(void)
 {
diff -Nur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/zapit/src/getservices.cpp neutrino-hd2-exp/src/zapit/src/getservices.cpp
--- neutrino-hd2-exp.org/src/zapit/src/getservices.cpp	2012-12-02 23:13:26.040785502 +0100
+++ neutrino-hd2-exp/src/zapit/src/getservices.cpp	2012-12-02 23:14:30.752788184 +0100
@@ -32,7 +32,7 @@
 #include <set>
 #include <sys/stat.h>
 #include <sys/vfs.h>
-
+#include <unistd.h>
 #include <errno.h>
 
 /* system */
diff -Nur '--exclude-from=diff-ignore' neutrino-hd2-exp.org/src/zapit/src/nit.cpp neutrino-hd2-exp/src/zapit/src/nit.cpp
--- neutrino-hd2-exp.org/src/zapit/src/nit.cpp	2012-12-02 23:13:26.044785502 +0100
+++ neutrino-hd2-exp/src/zapit/src/nit.cpp	2012-12-02 23:13:34.228785840 +0100
@@ -27,6 +27,7 @@
 #include <cerrno>
 #include <cstdio>
 #include <cstring>
+#include <unistd.h>
 
 /* system */
 #include <system/debug.h>
