diff -Naur '--exclude-from=diff-ignore' neutrino-hd.org/lib/libtuxtxt/tuxtxt.cpp neutrino-hd/lib/libtuxtxt/tuxtxt.cpp
--- neutrino-hd.org/lib/libtuxtxt/tuxtxt.cpp	2012-12-20 22:31:16.013918107 +0100
+++ neutrino-hd/lib/libtuxtxt/tuxtxt.cpp	2012-12-23 18:15:31.834869840 +0100
@@ -1725,13 +1725,19 @@
 	sscanf(cvs_revision, "%*s %s", versioninfo);
 	printf("TuxTxt %s\n", versioninfo);
 	printf("for 32bpp framebuffer\n");
-
+#ifdef MARTII
+        if ((fb=open("/dev/fb0", O_RDWR)) == -1)
+        {
+                perror("TuxTxt <open /dev/fb0>");
+                return 0;
+        }
+#else
         if ((fb=open("/dev/fb/0", O_RDWR)) == -1)
         {
                 perror("TuxTxt <open /dev/fb/0>");
                 return 0;
         }
-
+#endif
 	rc = _rc;
 	CFrameBuffer *fbp = CFrameBuffer::getInstance();
 #ifdef MARTII
diff -Naur '--exclude-from=diff-ignore' neutrino-hd.org/src/driver/framebuffer.h neutrino-hd/src/driver/framebuffer.h
--- neutrino-hd.org/src/driver/framebuffer.h	2012-12-20 22:31:16.021918106 +0100
+++ neutrino-hd/src/driver/framebuffer.h	2012-12-23 18:14:14.510866448 +0100
@@ -136,8 +136,10 @@
 
 #ifdef MARTII
 		void blitIcon(int src_width, int src_height, int fb_x, int fb_y, int width, int height);
-#endif
+		void init(const char * const fbDevice = "/dev/fb0");
+#else
 		void init(const char * const fbDevice = "/dev/fb/0");
+#endif
 		int setMode(unsigned int xRes, unsigned int yRes, unsigned int bpp);
 
 
