diff -Nur python-2.6.6.org/Modules/_ctypes/libffi/configure.ac python-2.6.6/Modules/_ctypes/libffi/configure.ac
--- python-2.6.6.org/Modules/_ctypes/libffi/configure.ac	2009-04-28 22:01:18.000000000 +0200
+++ python-2.6.6/Modules/_ctypes/libffi/configure.ac	2012-09-17 21:55:58.796875345 +0200
@@ -23,6 +23,7 @@
 m4_rename([_AC_ARG_VAR_PRECIOUS],[real_PRECIOUS])
 m4_define([_AC_ARG_VAR_PRECIOUS],[])
 AC_PROG_CC
+m4_undefine([_AC_ARG_VAR_PRECIOUS])
 m4_rename([real_PRECIOUS],[_AC_ARG_VAR_PRECIOUS])
 
 AC_SUBST(CFLAGS)
@@ -366,7 +367,7 @@
 
 AC_CONFIG_LINKS(include/ffitarget.h:src/$TARGETDIR/ffitarget.h)
 
-AC_CONFIG_FILES(include/ffi.h)
+AC_CONFIG_FILES(include/ffi.h include/Makefile Makefile)
 
 AC_CONFIG_LINKS(include/ffi_common.h:include/ffi_common.h)
 
diff -Nur python-2.6.6.org/Modules/_ctypes/libffi/fficonfig.py.in python-2.6.6/Modules/_ctypes/libffi/fficonfig.py.in
--- python-2.6.6.org/Modules/_ctypes/libffi/fficonfig.py.in	2010-01-16 19:05:15.000000000 +0100
+++ python-2.6.6/Modules/_ctypes/libffi/fficonfig.py.in	2012-09-17 21:55:58.768875345 +0200
@@ -30,4 +30,4 @@
 
 ffi_sources += ffi_platforms['@TARGET@']
 
-ffi_cflags = '@CFLAGS@'
+ffi_cflags = '@CFLAGS@ -DPIC'
diff -Nur python-2.6.6.org/Modules/_ctypes/libffi/Makefile.am python-2.6.6/Modules/_ctypes/libffi/Makefile.am
--- python-2.6.6.org/Modules/_ctypes/libffi/Makefile.am	2008-03-04 21:09:11.000000000 +0100
+++ python-2.6.6/Modules/_ctypes/libffi/Makefile.am	2012-09-17 21:55:58.796875345 +0200
@@ -2,7 +2,7 @@
 
 AUTOMAKE_OPTIONS = foreign subdir-objects
 
-SUBDIRS = include testsuite man
+SUBDIRS = include
 
 EXTRA_DIST = LICENSE ChangeLog.v1 ChangeLog.libgcj configure.host \
 	src/alpha/ffi.c src/alpha/osf.S src/alpha/ffitarget.h \
@@ -32,8 +32,6 @@
 	src/frv/ffi.c src/frv/eabi.S src/frv/ffitarget.h src/dlmalloc.c \
 	libtool-version ChangeLog.libffi
 
-info_TEXINFOS = doc/libffi.texi
-
 ## ################################################################
 
 ##

